backend/controllers/taskController.js

```diff
@@ -1,11 +1,27 @@
 const Createtodo = require("../models/taskModel");
 
+/* const userId = mongoose.Types.ObjectId(req.user.id);
+
+    const totalTasks = await Createtodo.countDocuments({ user: userId });
+
+    const paginatedTasks = await Createtodo.aggregate([
+      { $match: { user: userId } }, // Correctly match user ObjectId
+      { $skip: page * tasksPerPage },
+      { $limit: tasksPerPage },
+    ]);*/
 const getTasks = async (req, res) => {
   try {
-    const tasks = await Createtodo.find({ user: req.user.id });
+    const page = req.query.p || 0;
```

Comment:
what is p ? Please use the correct naming convention when use making the api easy for other developer to understand

Reply:
okay, will do so. here "p" is used for page no.